test/integration/02a-fund-bitcoind.spec.ts

```diff
@@ -65,6 +63,22 @@ it("funds bitcoind wallet", async () => {
   expect(balance).toBe(initialBitcoinWalletBalance + blockReward * numOfBlock)
 })
 
+// This new test fails afterFinished: checkIsBalanced()
```

Comment:
All tests still pass after I renamed the default wallet from "" (unnamed) to "default". But adding the test to create a second wallet ("hot") fails just after creation. Made a comment of this failure in the balance sheet: https://github.com/GaloyMoney/galoy/pull/317/files#diff-67ab19f16c472b30989f27f5755650158189756bbf68874767a0ff7196c50a05

@nicolasburtey @krtk6160 how should I proceed? Why is the "default" wallet not being considered for the balance sheet tests yet?

Reply:
not sure if you can change the default wallet. The current production environment use "" as default wallet

Reply:
@jotapea will need to dig deeper to understand why it's failing but some feedback on top of my head that could be relevant:
- since bitcoind 0.20 (or 0.21?) the behavior of bitcoind changed in regards to defauilt wallet and now no default wallet is being created automatically. the code has been udpated to accomodate this but IIRC it was a bit hacky
- we currently only use the bitcoin wallet in production is for the multisig that is created by specter. the other use of bitcoind is currently only for the test (maybe we should have 2 bitcoind instance to make this more explicit... one that is for the wallet we use, and another bitcoind that represents the "outside world")
- the test instantiated a separate wallet just because we need to have some BTC coming from somewhere. lnd is not getting the BTC directly from the coinbase rewards because the funds are not available immediately and I had issues with that: lnd typically doesn't expect to be the recipient of a coinbase rewards that has timelock restriction. (need to wait X blocks before the coinbase rewards is spendable)

Reply:
@nicolasburtey Renamed the second test from 02a-fund-bitcoind to 02a-fund-outside